flow/scene_update_context.cc

```diff
@@ -310,8 +312,8 @@ SceneUpdateContext::Frame::Frame(SceneUpdateContext& context,
 }
 
 SceneUpdateContext::Frame::~Frame() {
-  context().CreateFrame(std::move(entity_node_ptr()),
-                        std::move(shape_node_ptr()), rrect_, color_,
+  context().CreateFrame(std::move(entity_node()),
```

Comment:
Has https://fuchsia.atlassian.net/browse/SCN-268 been resolved? If so, I'm very glad that we can finally move the node directly without using so many pointers :)

Reply:
Yep, it has been resolved :)